components/automate-ui/src/app/entities/client-runs/client-runs.requests.ts

```diff
@@ -79,7 +79,8 @@ export class ClientRunsRequests {
       const params = this.formatFilters(filters).set('type', type).set('text', text);
       const url = `${CONFIG_MGMT_URL}/suggestions`;
 
-      return this.httpClient.get<Chicklet[]>(url, {params});
+      return this.httpClient.get<any[]>(url, {params}).pipe(map(
```

Comment:
Did you mean to loosen the type of the argument from `Chicklet` to `any` here?

Reply:
Yes, I did. The API is not returning a `Chicklet`. As I just noticed today a `Chicklet` is `{text:string, type:string}` and `{"score":1, "text": "some suggestion"}` is returned. We only want the text part. 
But I will create a new interface with text only.